public/editor/TopBar.tsx

```diff
@@ -33,8 +34,10 @@ export const TopBar = () => {
     gallerySectionConfigList,
     shuffleGenderSettingObject,
     shuffleRaceSettingObject,
+    shuffleOtherSettingObject,
     setShuffleGenderSettingObject,
     setShuffleRaceSettingObject,
+    setShuffleOtherSettingObject,
   } = stateStore;
   const genders: Gender[] = ["male", "female"];
```

Comment:
I think this was my fault originally, but should genders & races be imported from common as well?